lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/kotype/KoTypeDeclarationForKoFullyQualifiedNameProviderTest.kt

```diff
@@ -35,8 +35,8 @@ class KoTypeDeclarationForKoFullyQualifiedNameProviderTest {
         fun provideValues() = listOf(
```

Comment:
I'm not sure about this change. Maybe we should hardcode basic kotlin types? Add `kotlin.collections` for collections and `kotlin` for other basic types?

Reply:
This will not work because imports are missing in to cases
- kotlin standard library types are used (we could cover this)
- type is defined in the same package (it type would have the same name as kotlin standard library type, then FQN would be incorrect)